app/decorators/vm_or_template_decorator.rb

```diff
@@ -1,4 +1,6 @@
 class VmOrTemplateDecorator < MiqDecorator
+  include ERB::Util
```

Comment:
I'd not include the whole `ERB::Util`, just call it explicitly via `ERB::Util.h`, but maybe @martinpovolny has a different opinion.